lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart

```diff
@@ -24,6 +25,12 @@ class IoniaGiftCardDetailPage extends BasePage {
   IoniaGiftCardDetailPage(this.viewModel);
 
   final IoniaGiftCardDetailsViewModel viewModel;
+  double _brightness;
+
+  void increaseBrightness () async {
+    _brightness = await DeviceDisplayBrightness.getBrightness();
+    DeviceDisplayBrightness.setBrightness(1.0);
+  }
```

Comment:
Move this logic into view model, don't have BasePage is Stateless for now, so don't store mutated state there.

Reply:
done